src/baldr/CMakeLists.txt

```diff
@@ -69,24 +61,48 @@ valhalla_module(NAME baldr
     transitroute.cc
     transitschedule.cc
     transittransfer.cc
-    tz_alt.cpp
     laneconnectivity.cc
     verbal_text_formatter.cc
     verbal_text_formatter_us.cc
     verbal_text_formatter_us_co.cc
     verbal_text_formatter_us_tx.cc
-    verbal_text_formatter_factory.cc
+    verbal_text_formatter_factory.cc)
+
+#basic timezone stuff
+list(APPEND sources
+    ${CMAKE_CURRENT_BINARY_DIR}/date_time_africa.h
+    ${CMAKE_CURRENT_BINARY_DIR}/date_time_antarctica.h
+    ${CMAKE_CURRENT_BINARY_DIR}/date_time_asia.h
+    ${CMAKE_CURRENT_BINARY_DIR}/date_time_australasia.h
+    ${CMAKE_CURRENT_BINARY_DIR}/date_time_backward.h
+    ${CMAKE_CURRENT_BINARY_DIR}/date_time_etcetera.h
+    ${CMAKE_CURRENT_BINARY_DIR}/date_time_europe.h
+    ${CMAKE_CURRENT_BINARY_DIR}/date_time_leapseconds.h
+    ${CMAKE_CURRENT_BINARY_DIR}/date_time_northamerica.h
+    ${CMAKE_CURRENT_BINARY_DIR}/date_time_pacificnew.h
+    ${CMAKE_CURRENT_BINARY_DIR}/date_time_southamerica.h
+    ${CMAKE_CURRENT_BINARY_DIR}/date_time_systemv.h
+    ${CMAKE_CURRENT_BINARY_DIR}/date_time_windows_zones.h
+    tz_alt.cpp)
+
+#ios we have more work to make use of system tzdb
+if(APPLE)
```

Comment:
on ios we need two things. first we need to also add an include path because for some reason `ios.mm` does `#include "ios.h"` instead of `#include "data/ios.h"`. then we also need to compile the objective-c code as well. we could do this non-conditionally on linux but then we would need to require people to install the `gobjc++` package which seems like unneeded hassle. the question remains what will happen on mac builds of this. so we'll monitor it on CI.